pkg/common/resource_test.go

```diff
@@ -191,6 +191,19 @@ func TestParsePodResource(t *testing.T) {
 	assert.Equal(t, res.Resources[siCommon.CPU].GetValue(), int64(3000))
 	assert.Equal(t, res.Resources["nvidia.com/gpu"].GetValue(), int64(5))
 
+	// Add pod OverHead, only support CPU and Memory
```

Comment:
Hi @zhuqi-lucas , the getResource function would parse the resource which includes cpu, memory, gpu and other else.
Should we add filtering function to specific resource or a case which overhead contains different type of resources?

Reply:
Nothing in the KEP says that there is a limit on the type of resource that can be added. That means we need to follow the second option and extend the UT

Reply:
Thank you @wilfred-s and @0yukali0 for review, sure, addressed in latest PR.